Bug 1132672 - [RTL][FTE] Fix padding on Import buttons.

jfkthame opened the pull request.

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/1706499?v=3" alt="jfkthame" width="50px" height="50px" /> jfkthame](https://github.com/jfkthame) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=923a9a8f4d49)

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/1706499?v=3" alt="jfkthame" width="50px" height="50px" /> jfkthame](https://github.com/jfkthame) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=cfcb3e5d0438)

jfkthame committed:
Bug 1132672 - [RTL][FTE] Fix padding on Import buttons.
It looks like this was added specifically to compensate for the broken
platform behavior (bug 1130231), so we can remove it now.

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/1706499?v=3" alt="jfkthame" width="50px" height="50px" /> jfkthame](https://github.com/jfkthame) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=acfdfe36f2a3)
